examples/test-app/grails-app/conf/application.yml

```diff
@@ -80,6 +80,12 @@ gorm:
     tools:
         mango:
             criteriaKeyName: criteria
+            defaultQuickSearch:
```

Comment:
@alexeyzvegintcev this can also be represented as ['name', 'num', ec....] which I think is cleaner in this case